Tests/SnapshotTests.swift

```diff
@@ -0,0 +1,66 @@
+// Created by Cal Stephens on 12/8/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+@testable import Lottie
```

Comment:
nit: our style guide recommends splitting these out below the other imports with a newline

Reply:
TIL, didn't know about this rule

Reply:
Aha, the linter was passing because it wasn't set up to run on this folder. Fixed, thanks for catching this.